src/server/session.ts

```diff
@@ -304,7 +304,9 @@ function allEditsBeforePos(edits: readonly TextChange[], pos: number): boolean {
 // The var assignment ensures that even though CommandTypes are a const enum
 // we want to ensure the value is maintained in the out since the file is
 // built using --preseveConstEnum.
+/** @deprecated use ts.server.protocol.CommandTypes */
 export type CommandNames = protocol.CommandTypes;
+/** @deprecated use ts.server.protocol.CommandTypes */
 export const CommandNames = (protocol as any).CommandTypes;

```

Comment:
I wonder if these could have just been declared as `import CommandNames = protocol.Commandtypes` in the first place. It probably would preserve the enum inlining issue you commented on, but at this point, I wouldn't worry about it.

Reply:
That's true, this could be `export import CommandNames = protocol.CommandTypes`; I'll have to see if it bundles alright as that does help a bit.

But, I do think that if we're switching to direct imports, no reason not to use the real deal.

Reply:
Hah, so this is syntax I never bothered to implement in the dts bundler since I removed all instances during the conversion. Oops.